db/rules.go

```diff
@@ -180,9 +182,27 @@ func (nh *NatsHandler) ruleRunActions(nc *natsgo.Conn, r *data.Rule, triggerNode
 				return err
 			}
 		case data.PointValueActionPlayAudio:
+			f, err := os.Open(a.PointFilePath)
+			if err != nil {
+				log.Fatal(err)
+			}
+			defer f.Close()
+
+			d := wav.NewDecoder(f)
+			d.ReadInfo()
+
+			format := d.Format()
+
+			if format.SampleRate < 8000 {
+				log.Println("Rule action: invalid wave file sample rate: ", format.SampleRate)
+				continue
+			}
+
 			channelNum := strconv.Itoa(a.PointChannel)
+			sampleRate := strconv.Itoa(format.SampleRate)
```

Comment:
ended up determining sample rate of wave file as `speaker-test` seems to need this if its not set to some default